client/reader/following-manage/sites-window-scroller.jsx

```diff
@@ -55,37 +61,62 @@ class SitesWindowScroller extends Component {
 		this.listRef = list;
 	}
 
-	handleResize = debounce( () => {
+	handleResize = debounce( () => this.clearListCaches(), 50 );
+
+	clearListCaches = () => {
 		this.heightCache.clearAll();
 		defer( () => this.listRef && this.listRef.recomputeRowHeights( 0 ) );
-	}, 50, { trailing: true } );
+	}
+
+	isRowLoaded = ( { index } ) => {
+		return !! this.props.sites[ index ];
+	}
+
+	// technically this function should return a promise that only resolves when the data is fetched.
+	// initially I had created a promise that would setInterval and see if the startIndex
+	// exists in sites, and if so the resolve. It was super hacky, and its muchs simpler to just fake that it instantly
+	// returns
+	// TODO: does a util function exist that return waitFor( thingToExistInStateTree )? that would be perfect.
+	loadMoreRows = ( { startIndex } ) => {
+		this.props.fetchNextPage && this.props.fetchNextPage( startIndex );
+		return Promise.resolve();
```

Comment:
I wrote that exact thing at first...then deleted it.  Totally not worth it.  Lying to RV seems cleaner to me